pkg/bootstrap/precheck/tasks.go

```diff
@@ -62,10 +70,7 @@ func (n *NodePreCheck) Execute(runtime connector.Runtime) error {
 		if err != nil || strings.Contains(res, "not found") {
 			results[software] = ""
 		} else {
-			if strings.HasPrefix(cmd, "which") {
```

Comment:
Sorry, I don't know why should modify here. Here, the result of `docker` and `containerd` will show their version.